docs/_config.yml

```diff
@@ -22,7 +22,7 @@ include:
 SPARK_VERSION: 4.0.0-SNAPSHOT
 SPARK_VERSION_SHORT: 4.0.0
 SCALA_BINARY_VERSION: "2.13"
-SCALA_VERSION: "2.13.11"
+SCALA_VERSION: "2.13.12"
```

Comment:
The change of `docs/_config.yml` was forgotten in https://github.com/apache/spark/pull/43185.